lightning/src/onion_message/messenger.rs

```diff
@@ -572,6 +572,13 @@ where
 		// recipient's node_id.
 		const MIN_PEER_CHANNELS: usize = 3;
 
+		// Add a random number (1 to 5) of dummy hops to each non-compact blinded path
+		// to make it harder to infer the recipient's position.
```

Comment:
Document reason why compact paths can't have dummy hops?

Reply:
Expanded the docs to document why we don't add dummy hops to compact paths in [pr3728.06](https://github.com/shaavan/rust-lightning/commits/pr3728.06). Thanks!